File: Iserv.Niis.Model/Models/ExpertSearch/ExpertSearchSimilarDto.cs
using Iserv.Niis.Domain.Helpers;
using System;

namespace Iserv.Niis.Model.Models.ExpertSearch
{
    public class ExpertSearchSimilarDto
    {
        public int Id { get; set; }
        public DateTimeOffset DateCreate { get; set; }
        public int RequestId { get; set; }
        public Owner.Type OwnerType { get; set; }
        public int? SimilarRequestId { get; set; }
        public Domain.Entities.Request.Request SimilarRequest { get; set; }
        public int? SimilarProtectionDocId { get; set; }
        public Domain.Entities.ProtectionDoc.ProtectionDoc SimilarProtectionDoc { get; set; }
        public int ImageSimilarity { get; set; }
        public int PhonSimilarity { get; set; }
        public int SemSimilarity { get; set; }
        public string ProtectionDocFormula { get; set; }
        public string ProtectionDocCategory { get; set; }
    }
}

File: Iserv.Niis.Angular/src/app/expert-search/models/expert-search-similar-dto.ts
import { OwnerType } from '../../shared/services/models/owner-type.enum';
export class ExpertSearchSimilarDto {
  id: number;
  dateCreate: string;
  requestId: number;
  ownerType: OwnerType;
  similarRequestId: number;
  similarProtectionDocId: number;
  imageSimilarity: string;
  phonSimilarity: string;
  semSimilarity: string;

  constructor(init?: Partial<ExpertSearchSimilarDto>) {
    Object.assign(this, init);
  }
  protectionDocFormula: string;
  protectionDocCategory: string;
}



